### PR TITLE
#976 - azure sign in tool now prompts for username with each login

### PR DESCRIPTION
**Pull request by justinwilaby**
#976 

Fixed

### CODE REVIEW

File: packages/app/main/src/services/azureAuthWorkflowService.ts [hunk 1]
```diff
@@ -92,8 +92,8 @@ export class AzureAuthWorkflowService {
         const values = uri.substring(idx + 1).split('&');
         const len = values.length;
         for (let i = 0; i < len; i++) {
-          const [key, value] = values[i].split(/[=]/);
-          result[key] = value;
+          const [ key, value ] = values[ i ].split(/[=]/);
```
Comment (carlosscastro) [inline packages/app/main/src/services/azureAuthWorkflowService.ts]: This feels weird, I prefer values[i]. Sure you want this edit?

File: packages/app/main/src/services/azureAuthWorkflowService.ts [hunk 2]
```diff
@@ -92,8 +92,8 @@ export class AzureAuthWorkflowService {
...
+          result[ key ] = value;
```
Comment (carlosscastro) [inline packages/app/main/src/services/azureAuthWorkflowService.ts]: Same here, not a fan of this, not sure if it is a standard in js but it looks like an editor automated thing

File: packages/app/main/src/services/azureAuthWorkflowService.ts [hunk 3]
```diff
@@ -247,7 +246,7 @@ export class AzureAuthWorkflowService {
   }
 
   private static async validateJWT(token: string): Promise<boolean> {
-    const [header] = token.split('.');
+    const [ header ] = token.split('.');
```
Comment (carlosscastro) [inline packages/app/main/src/services/azureAuthWorkflowService.ts]: same here, is this on purpose?

Comment (justinwilaby) [reply packages/app/main/src/services/azureAuthWorkflowService.ts]: Auto format - rules might have changed unintentionally.

**Pull request merged**